cupyx/scipy/signal/signaltools.py

```diff
@@ -0,0 +1,81 @@
+import math
+
+import cupy
+
+
+def _numeric_arrays(arrays, kinds='buifc'):
+    if type(arrays) == cupy.ndarray:
+        return arrays.dtype.kind in kinds
+    for array in arrays:
+        if array.dtype.kind not in kinds:
+            return False
+    return True
+
+
+def choose_conv_method(in1, in2, mode='full'):
+    """Find the fastest convolution/correlation method.
+
+    Args:
+        in1 (cupy.ndarray): first input.
+        in2 (cupy.ndarray): second input.
+        mode (str, optional): `valid`, `same`, `full`
+
+    Returns:
+        str: A string indicating which convolution method is fastest,
+         either ‘direct’ or ‘fft’.
```

Comment:
Correct indentation and quotations.
```suggestion
        either ``direct`` or ``fft1``.
```